helper.js

```diff
@@ -182,6 +181,7 @@ HELPER.compile = (() => {
 	const Sassify = ( scss, css ) => {
 		const compiled = Sass.renderSync({
 			file: scss,
+			indentType: 'tab',
```

Comment:
No it won't but this bit of the helper will be replaced by pancake once it's done I assume so I just want them all to have the same settings for now.

Also: Tabs are literally the one and only semantic element for indentation. LITERALLY!